openssl/src/ssl/mod.rs

```diff
@@ -613,6 +613,24 @@ impl SslContext {
     }
 
     #[allow(non_snake_case)]
+    /// Advertise the CAs from a file as being accepted for client certificates.
+    ///
+    /// This does not actually make the certificates trusted. You'll still need to call set_CA_file
+    /// for that.
+    pub fn advertise_client_CA_list<P: AsRef<Path>>(&mut self, file: P) -> Result<(), SslError> {
```

Comment:
Can this name get changed to `set_client_CA_list` to match the OpenSSL function name?